modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,157 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to start event streaming in the cloud: You can start streaming in 30 seconds. 
+
+== What is Redpanda Serverless?
+
+Redpanda Serverless is a managed streaming service (Kafka API) that completely abstracts users from scaling and operational concerns, and you only pay for what you use. It provides data isolation and data residency:
+
+* Your data is hosted on Redpanda Cloud clusters shared among multiple tenants. Other tenants never access your data.
+* Your data is stored in your Redpanda Cloud account, in the region you select. Data never leaves your account.
+
+=== Deployment and networking
+
+Redpanda Serverless currently is available in select regions in AWS. We expect your applications to be deployed in the same AWS region. For best performance, select the region closest to your applications. Serverless is not guaranteed to be pinned to a particular availability zone within that region.
+
+Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Redpanda Dedicated clusters for private networking.
+
+=== Limitations
+
+Each Serverless cluster has the following limits:
+
+* Ingress: up to 10 MBps, 0.5 MBps guaranteed
+* Egress: up to 30 MBps, 1.5 MBps guaranteed
+* Partitions: 100
+* Max message size: 20 MB
+* Max retention: unlimited
+* Storage: unlimited
+
+=== Serverless vs Dedicated and BYOC
+
+Redpanda Serverless is a good fit for the following:
+
+* Starter and growning workloads
+* Projects that don't saturate a Dedicated cluster all the time
+* Spiky workloads (dev environments, systems that only occasionally get busy, or workloads that come and go)
+* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
+
+Because a Serverless cluster is created in under 1 second, you can surface it in your applications (for example, for tenant isolation).
+
+If necessary, workloads can be migrated from Redpanda Serverless to Redpanda Dedicated or BYOC clusters. Consider Dedicated or BYOC for the following:
+
+* Control over cluster settings
+* Control over the deployment
+* Private networking
+* Workloads with consistently-high throughput
+
+== Free trial 
+
+Try our Beta release of Redpanda Serverless to quickly interact with a Serverless cluster, connect to your application, and stream data from your laptop. To ask questions, give feedback, or extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
```

Comment:
> Try our Beta release of Redpanda Serverless to quickly interact with a Serverless cluster, connect to your application, and stream data from your laptop.

Please replace with:

Start using Redpanda Serverless Beta by joining the free trial!